File: 2021/2021-09/2021-9-21 58.最后一个单词的长度/LengthOfLastWord.cs
public class Solution {
    public int LengthOfLastWord(string s) 
    {
        var result = 0;
        for (var i = s.Length - 1; i >= 0; i--)
        {
            if ( s[i] == ' ')
            {
                if (result == 0)
                {
                    continue;
                }
                else
                {
                    break;
                }
            }
            result++;
        }
        return result;
    }
}

File: 2021/2021-09/2021-9-21 58.最后一个单词的长度/lengthOfLastWord.go
package main

func main() {
	s := "   fly me   to   the moon  "
	lengthOfLastWord(s)
}

func lengthOfLastWord(s string) int {
	l := len(s) - 1
	var res int
	for i := l; i >= 0; i-- {
		if string(s[i]) != " " {
			res++
		} else if res != 0 {
			return res
		}
	}
	return res
}
